draco/analysis/transform.py

```diff
@@ -485,62 +484,71 @@ def process(self, sstream):
         if self.telescope is not None:
             mmax = self.telescope.mmax
         else:
-            mmax = None
-
-        # Construct the array of m-modes
-        marray = _make_marray(sstream.vis[:], mmax)
-        marray = mpiarray.MPIArray.wrap(marray[:], axis=freq_axis, comm=sstream.comm)
+            mmax = (sstream.vis.shape[-1] // 2) - 1
```

Comment:
I think this needs to be:
```suggestion
            mmax = sstream.vis.shape[-1] // 2
```

Otherwise the round-trip m-mode transform:
```
    - type: draco.analysis.transform.MModeTransform
      in: sstream
      out: mmode

    - type: draco.analysis.transform.MModeInverseTransform
      in: mmode
      out: rt_sstream
```
results in an RA axis with size N_{RA} -1 when N_{RA} is even and N_{RA} - 2 when N_{RA} is odd.

Reply:
Thanks! I'll fix that one.